python/test/test_table_from_numpy.py

```diff
@@ -84,10 +84,84 @@
 
 print(dict3)
 
-print(cn_tb5.schema)
-
 print(cn_tb5.column_names)
 
 print(ar_tb2)
 
-print(cn_tb5.to_numpy())
\ No newline at end of file
+print(cn_tb5.to_numpy())
+
+## Aggregate Sum
+
+cn_tb6 = cn_tb5.sum('col1')
+
+cn_tb6.show()
+
+cn_tb7 = cn_tb5.sum(0)
+
+## Aggregate Count
+
+cn_tb8 = cn_tb5.count('col1')
+
+cn_tb8.show()
+
+cn_tb9 = cn_tb5.count(0)
+
+cn_tb9.show()
+
+## Aggregate Min
+
+cn_tb10 = cn_tb5.min('col1')
+
+cn_tb10.show()
+
+cn_tb11 = cn_tb5.min(0)
+
+cn_tb11.show()
+
+## Aggregate Max
+
+cn_tb12 = cn_tb5.max('col1')
+
+cn_tb12.show()
+
+cn_tb13 = cn_tb5.max(0)
+
+cn_tb13.show()
+
+from pycylon.data.aggregates import AggregationOp
+
+op1 = AggregationOp.SUM
+
+assert (op1 == AggregationOp.SUM)
+
+print(op1.name)
+
+dict3 = {'col1': [1, 2, 3, 4, 5, 1, 3, 6, 8, 1, 9, 10], 'col2': [2, 4, 0, 1, 5, 6, 8, 1, 3, 4, 0,
+                                                                 1]}
+
+cn_tb14: cn.Table = cn.Table.from_pydict(ctx, dict3)
+
+cn_tb14.groupby(0, [0], [AggregationOp.COUNT])
+
+cn_tb14.show()
```

Comment:
lets use asserts to actually check for the correct value

Reply:
the test bench will be separately added for all Python use cases.